examples/05_glm_second_level/plot_second_level_one_sample_test.py

```diff
@@ -94,47 +115,90 @@
 p_val = second_level_model.compute_contrast(output_type='p_value')
 n_voxels = np.sum(get_data(second_level_model.masker_.mask_img_))
 # Correcting the p-values for multiple testing and taking negative logarithm
-neg_log_pval = math_img("-np.log10(np.minimum(1, img * {}))"
-                        .format(str(n_voxels)),
-                        img=p_val)
-
-###########################################################################
-# Let us plot the (corrected) negative log p-values for the parametric test.
-cut_coords = [0]
-# Since we are plotting negative log p-values and using a threshold equal to 1,
-# it corresponds to corrected p-values lower than 10%, meaning that there is
-# less than 10% probability to make a single false discovery (90% chance that
-# we make no false discovery at all).  This threshold is much more conservative
-# than the previous one.
-threshold = 1
-title = ('Group left-right button press: \n'
-         'parametric test (FWER < 10%)')
-display = plotting.plot_glass_brain(
-    neg_log_pval, colorbar=True, display_mode='z', plot_abs=False, vmax=3,
-    cut_coords=cut_coords, threshold=threshold, title=title)
-plotting.show()
+neg_log_pval = math_img(
+    '-np.log10(np.minimum(1, img * {}))'.format(str(n_voxels)),
+    img=p_val,
+)
 
-###########################################################################
+###############################################################################
 # Now, we compute the (corrected) p-values with a permutation test.
+#
+# .. important::
+#   In this example, ``threshold`` is set to 0.001, which enables
```

Comment:
Ah I see (reading the function docstring further down) ! Is there a reason not to first call `non_parametric_inference` with `threshold` set to `None` and then again as is done here (with a provided `threshold` value) ?

Reply:
Do you mean just to show how `non_parametric_inference` would work with and without cluster-level inference? If we ran it without cluster-level inference, it would take some additional time to run the voxel-level-only permutations, and we get those for free when we use `threshold`. Sorry if I'm misunderstanding your question.

Reply:
No, that's my question :) I think from a UX standpoint this isn't something that's obvious in the example flow. Maybe I'm missing something from the non-rendered files -- I'll try rendering to see if that helps me understand the logical progression a bit more easily.

Reply:
To clarify my question : When we make a general recommendation for how users can correct their p-values _without_ cluster-level inference, will we point them to [this math_img code block](https://github.com/nilearn/nilearn/pull/3181/files/1e7bc40628b701735bd58b97082e1ba6493d314a#diff-ea11adc25ebe65ff700093bf22dc77c490bfce5298bc8d9a26fa9216af068d32R118-R121) ? Or would we point them to the `non_parametric_inference` function and tell them to make sure that `threshold` is set to `None` ?

Reply:
I think the code block you linked to is for the parametric approach instead. I guess I'd point them to the function... but I could add an admonition here about running just voxel-level correction?

Reply:
I added an admonition in c74f809. WDYT?